sqlparser/lib/src/engine/module/fts5.dart

```diff
@@ -28,6 +28,8 @@ class _Fts5Module extends Module {
     return _Fts5Table(
       name: stmt.tableName,
       columns: [
+        if (stmt.argumentContent.any((arg) => arg.startsWith('content_rowid')))
+          TableColumn('rowid', const ResolvedType(type: BasicType.int)),
```

Comment:
I'm not sure why the `rowid` only exists when there's a `content_rowid` option? Shouldn't it be exposed whenever we have a content table in general?

Reply:
At first I got the documentation wrong, I double checked and validated that `content_rowid` is actually optional and defaults to 'rowid'